src/test/java/ymfc/parser/ParserTest.java

```diff
@@ -167,7 +169,17 @@ void parseCommand_addRecipeCommand_success(Recipe recipe, String command) {
 
     })
     void parseCommand_addRecipeCommand_invalidArgumentsExceptionThrown(String command) {
-        assertThrows(InvalidArgumentException.class, () -> parseCommand(command));
+        RecipeList recipes = new RecipeList();
+
+        ArrayList<String> pastaIngredients = new ArrayList<>();
+        pastaIngredients.add("Pasta");
+        ArrayList<String> pastaSteps = new ArrayList<>();
+        pastaSteps.add("Boil pasta in water.");
+
+        Recipe pastaRecipe = new Recipe("Pasta", pastaIngredients, pastaSteps, 2);
+        recipes.addRecipe(pastaRecipe);
```

Comment:
I don't think this test unit of `addRecipeCommand` will need to manually adding a filled `recipeList`. Just an empty `RecipeList` will do the job since the `add` command doesn't required a `recipeList` to be non-empty.

An alternative way is to use the `@Before` (`@BeforeEach`, `@BeforeAll`) tag provided by JUnit. Maybe [this](https://junit.org/junit5/docs/current/user-guide/#writing-tests-annotations) will help.

Reply:
Yea you're right, it can be left as an empty list. Will remove these lines.